### DISCUSSION TITLE
Contribution Guidelines

### DISCUSSION THREAD

**Discussion by rhughes42** [linkarkitektur/SpeckleLCA#24] [General]
@fabianlinkflink @pfmephisto I just saw the doc now, I'm going to create a fork and get started there, maybe make a few suggestions. @pfmephisto is that the latest version or have you got something local you're not sharing? 🧑‍💻

**Comment by rhughes42** [comment]
https://github.com/specklesystems/speckle-sharp/blob/main/.github/CONTRIBUTING.md I like their bug and cosmetic patch differentiation etc.

**Comment by pfmephisto** [comment]
@rhughes42 No, that is the latest one.
I have started an effort to port the project to vue3/4 etc. But even that branch is public.

**Reply by rhughes42** [reply]
Do you know what you need to do to port? Are there any unknown unknowns regarding: how Vue is hooked into the rest of the tooling, or should it be straightforward? If we make a project out of that, I can help. Also, we could make a modular Vue architecture whereby we inject modules at app load time, which we can even do with Vue 2.

**Reply by pfmephisto** [reply]
I think the easiest might just be to start of with a fresh clean project and the copy the files over and modify them.
But I think I need to look at some of the differences. I can see that they no longer suggest vuex as store but Pinia instead.
But it might just be a drop in replacement.
